lldb/test/API/tools/lldb-dap/step/TestDAP_step.py

```diff
@@ -83,3 +83,40 @@ def test_step(self):
 
                     # only step one thread that is at the breakpoint and stop
                     break
+
+    def test_step_over(self):
+        """
+        Test stepping over when the program counter is in another file.
+        """
+        program = self.getBuildArtifact("a.out")
+        self.build_and_launch(program)
+        source = "main.cpp"
+        breakpoint1_line = line_number(source, "// breakpoint 2")
```

Comment:
`breakpoint2_line` or just `breakpoint_line` since we only have 1 in this test? Its a bit odd to call this `breakpoint1_line` when it in fact points to `"// breakpoint 2"`.